### ISSUE TITLE
Style/SymbolProc auto-correct breaking syntax

### ISSUE THREAD

**Issue by NobodysNightmare** [bbatsov/rubocop#2588]
## Reproduction

````
$ ruby -v
ruby 2.2.3p173 (2015-08-18 revision 51636) [x86_64-linux]
$ rubocop -v
0.35.1
````

Using the following test file (dubbed `rubocop_test.rb`):

````ruby
def mail(to:, subject:)
  yield to
end

mail(
      to: 'foo',
      subject: 'bar',
    ) { |format| format.text }
````

Run `rubocop -a rubocop_test.rb`

## Expected

A valid ruby file, with fewer offenes.

## Actual

The syntax gets messed up by the `Style/SymbolProc` rule:

````ruby
def mail(to:, subject:)
  yield to
end

mail(
  to: 'foo',
  subject: 'bar', &:text,
)
````

The extra comma after the `&:text` parameter is breaking the syntax.

**Comment by alexdowad** [comment]
> The extra comma after the &:text parameter is breaking the syntax.

The extra comma is actually the one which was already there, after `subject: 'bar'`.

Thinking about how to resolve this...

**Comment by alexdowad** [comment]
I have an idea.

**Comment by NobodysNightmare** [comment]
Wow...

I totally copy and pasted that example out of production code and didn't notice the comma after `subject: 'bar'`...
Now I am wondering why it was valid before, but I guess ruby is forgiving at times...

**Comment by alexdowad** [comment]
Just pushing a fix to my open PR.

**Comment by NobodysNightmare** [comment]
@alexdowad Wow, that is what I call a quick response!

Thanks for your efforts :+1: (and bonus points for the title of your PR ^^)
